Cargo.lock

```diff
@@ -7995,6 +7872,15 @@ dependencies = [
  "windows-targets 0.48.0",
 ]
 
+[[package]]
+name = "windows-sys"
+version = "0.52.0"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "282be5f36a8ce781fad8c8ae18fa3f9beff57ec1b52cb3de0789201425d9a33d"
+dependencies = [
+ "windows-targets 0.52.6",
+]
```

Comment:
Do we really need this version? can't we rely on `0.59.0` which we already have.

Reply:
I guess this https://github.com/rust-lang/socket2/blob/36dec54c30c5ca0c711709b9e1f03b8cb8d155fa/Cargo.toml#L57 sets a specific version so maybe we can't remove this for now

Reply:
done